app/models/itsaStatus/StatusDetail.scala

```diff
@@ -49,4 +49,23 @@ object ITSAStatus extends Enumeration {
 
   implicit val itsaStatusReads: Reads[ITSAStatus] = Reads.enumNameReads(ITSAStatus)
   implicit val itsaStatusWrite: Writes[ITSAStatus] = Writes.enumNameWrites
+
+  def statusToString(status: ITSAStatus): String =
+    status match {
+      case ITSAStatus.NoStatus  => "U"
+      case ITSAStatus.Voluntary => "V"
+      case ITSAStatus.Annual    => "A"
+      case ITSAStatus.Mandated  => "M"
+      // This will be validated earlier on in a future ticket
+      case _ => throw new RuntimeException("Unexpected status")
+    }
+
+  def stringToStatus(status: String): ITSAStatus.Value =
+    status match {
+      case "U" => ITSAStatus.NoStatus
+      case "V" => ITSAStatus.Voluntary
+      case "A" => ITSAStatus.Annual
+      case "M" => ITSAStatus.Mandated
+      case _ => throw new RuntimeException("Unexpected status")
+    }
```

Comment:
I don't think this is the right place to put these conversion methods. These are used to convert between the ITSAStatus and the Opt Out/In Journey repository. By being placed here on the model, makes them more central and would encourage uses in other contexts, which would not be idea. It might be more appropriate to have them as helpers in the repositories package, though would be happy to discuss other possibilities.

Reply:
I still think this is the right and even perfect place for it, again saves clicking around packages to find stuff, but in any case will move them to helper object in repositories

Reply:
done